WordPressKitTests/Mock Data/me-settings-close-account-failure.json

```diff
@@ -0,0 +1,4 @@
+{
+    "error": "active-subscriptions"
+    "message": "This user account cannot be closed while it has active subscriptions."
+}
```

Comment:
As the GitHub syntax highlighting suggests, there's an error in this JSON, with a missing comma on line 2.

Not sure if that's intentional (e.g. to use it as mock data to test invalid JSON returned by the server) but I don't think so? So might be worth investigating and fixing in a future PR.